frontend/src/components/indicators/AddEditIndicator.js

```diff
@@ -67,8 +67,8 @@ export default function AddEditIndicator() {
   const attribute2Compass = {
     name: 'cids:hasName',
     description: 'cids:hasDescription',
-    organization: 'cids:Organization',
-    indicator: 'cids:Indicator',
+    organization: 'cids:forOrganization',
+    indicator: 'cids:forIndicator',
```

Comment:
The attribute2Compass should not be the same as the IndicatorReport's. The entries in attribute2Compass should correspond with the form on that page.